--- schema_old.rs	2024-01-29 15:27:13.298391417 +0000
+++ schema.rs	2024-01-29 15:28:30.420393299 +0000
@@ -15,20 +15,12 @@

     #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "fertility"))]
     pub struct Fertility;

     #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "geography"))]
-    pub struct Geography;
-
-    #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "geometry"))]
-    pub struct Geometry;
-
-    #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "growth_rate"))]
     pub struct GrowthRate;

     #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "herbaceous_or_woody"))]
     pub struct HerbaceousOrWoody;
@@ -165,16 +157,15 @@
         is_alternative -> Bool,
     }
 }

 diesel::table! {
     use postgis_diesel::sql_types::Geography;
+    use postgis_diesel::sql_types::Geometry;
     use diesel::sql_types::*;
     use super::sql_types::PrivacyOption;
-    use super::sql_types::Geography;
-    use super::sql_types::Geometry;

     maps (id) {
         id -> Int4,
         name -> Text,
         deletion_date -> Nullable<Date>,
         last_visit -> Nullable<Date>,
