--- src/schema.rs	2024-04-23 23:15:12.521217867 +0200
+++ src/schema_tmp.rs	2024-04-23 23:18:06.133856878 +0200
@@ -19,20 +19,12 @@

     #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "fertility"))]
     pub struct Fertility;

     #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "geography"))]
-    pub struct Geography;
-
-    #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "geometry"))]
-    pub struct Geometry;
-
-    #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "growth_rate"))]
     pub struct GrowthRate;

     #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "herbaceous_or_woody"))]
     pub struct HerbaceousOrWoody;
@@ -185,16 +177,15 @@
         order_index -> Int4,
     }
 }

 diesel::table! {
     use postgis_diesel::sql_types::Geography;
+    use postgis_diesel::sql_types::Geometry;
     use diesel::sql_types::*;
     use super::sql_types::PrivacyOption;
-    use super::sql_types::Geography;
-    use super::sql_types::Geometry;

     maps (id) {
         id -> Int4,
         name -> Text,
         creation_date -> Date,
         deletion_date -> Nullable<Date>,
