--- src/schema.rs	2024-02-19 18:13:43.582300382 +0000
+++ src/schema_tmp.rs	2024-02-19 18:12:36.849610756 +0000
@@ -15,20 +15,12 @@

     #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "fertility"))]
     pub struct Fertility;

     #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "geography"))]
-    pub struct Geography;
-
-    #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "geometry"))]
-    pub struct Geometry;
-
-    #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "growth_rate"))]
     pub struct GrowthRate;

     #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "herbaceous_or_woody"))]
     pub struct HerbaceousOrWoody;
@@ -162,14 +154,12 @@
 }

 diesel::table! {
     use postgis_diesel::sql_types::Geography;
     use postgis_diesel::sql_types::Geometry;
     use super::sql_types::PrivacyOption;
-    use super::sql_types::Geography;
-    use super::sql_types::Geometry;

     maps (id) {
         id -> Int4,
         name -> Text,
         creation_date -> Date,
         deletion_date -> Nullable<Date>,
