diff --git a/backend/src/schema.rs b/backend/src/schema.rs 2023-07-20
index 54f26f46..68427977 100644
--- a/backend/src/schema.rs
+++ b/backend/src/schema.rs
@@ -10,20 +10,12 @@ pub mod sql_types {
     pub struct ExternalSource;

     #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "fertility"))]
     pub struct Fertility;

-    #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "geography"))]
-    pub struct Geography;
-
-    #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "geometry"))]
-    pub struct Geometry;
-
     #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "growth_rate"))]
     pub struct GrowthRate;

     #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "herbaceous_or_woody"))]
@@ -100,16 +92,15 @@ diesel::table! {
         is_alternative -> Bool,
     }
 }

 diesel::table! {
     use postgis_diesel::sql_types::Geography;
+    use postgis_diesel::sql_types::Geometry;
     use diesel::sql_types::*;
     use super::sql_types::PrivacyOption;
-    use super::sql_types::Geography;
-    use super::sql_types::Geometry;

     maps (id) {
         id -> Int4,
         name -> Text,
         creation_date -> Date,
         deletion_date -> Nullable<Date>,
